README.md

```diff
@@ -11,9 +11,8 @@ This is a web-app agent for the raspberry pi. It allows you to interact easily w
 ## Devs only
 
 ### Pre-requisites
-
+- [pre-commit](https://pre-commit.com/) (pip install -U pre-commit)
```

Comment:
I'm sure there's some npm utils we can use, I don't wanna have python tools on top of node and golang

Reply:
could that work instead
https://www.npmjs.com/package/pre-commit

Reply:
and this for golang
https://github.com/dnephin/pre-commit-golang